Source/non_ip/apple_relay.conf

```diff
@@ -0,0 +1,10 @@
+# $ meta_title Sukka's Ruleset - Apple Relay
+# $ meta_description This file contains domains for Apple Relay, hosted by Apple, Inc.
+
+# >> Apple Relay
+DOMAIN-SUFFIX,apple-relay.fastly-edge.com
+DOMAIN-SUFFIX,apple-relay.cloudflare.com
+DOMAIN-SUFFIX,apple-relay.apple.com
+DOMAIN-SUFFIX,cp2.cloudflare.com
+DOMAIN-SUFFIX,cp3.cloudflare.com
+DOMAIN-SUFFIX,cp4.cloudflare.com
```

Comment:
Not specifically for Apple. Many Cloudflare customers also use these domains.

Reply:
https://blog.cloudflare.com/stronger-than-a-promise-proving-oblivious-http-privacy-properties/
https://docs.fastly.com/products/oblivious-http-relay

The OHTTP Relay is designed to hide users' IP/Geolocation (and other PI & PII) from the service providers. Why do you want to specify a special policy for them?